.github/workflows/scripts/test_scripts/run_spec_decode_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
We can remove this file for now because we have not set up the CI

Reply:
removed